TypeScript.tmLanguage

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```

Comment:
Yes, I change the file `TypeScript.YAML-tmLanguage`. Then I generated the XML `TypeScript.tmLanguage` with the [AAAPackageDev](https://github.com/SublimeText/PackageDev) plugin in Sublime Text.